src/flattenizer.ts

```diff
@@ -26,19 +45,22 @@ export const flatten = (unflattened: object, delimiter: string = '.'): object =>
     throw new TypeError('delimiter must be a string');
   }
 
-  const flattened = {};
-
-  Object.entries(unflattened).forEach(([key, value]) => {
+  const flattened: A = Object.keys(unflattened).reduce((acc, key) => {
```

Comment:
💡 `acc` is a typical name for a reduce's accumulator, so any developer that knows `reduce` will immediately know what it's doing, but it might be worth considering a semantic name for what this represents. I tried for a little bit and couldn't think of something that is distinct enough from `flatObject` below to like more than `acc`.